.travis.yml

```diff
@@ -9,10 +9,22 @@ jobs:
     - name: "Cookstyle"
       script:
         - bundle exec cookstyle -f fuubar
+
+    # Selected long-running tests are at the top to minimize Travis wall time
+    - name: "Test Kitchen (web::backend)"
+      script:
+        - bundle exec kitchen test web-backend-ubuntu-1804
+    - name: "Test Kitchen (web::frontend)"
+      script:
+        - bundle exec kitchen test web-frontend-ubuntu-1804
+    - name: "Test Kitchen (web::rails)"
+      script:
+        - bundle exec kitchen test web-rails-ubuntu-1804
     - name: "Test Kitchen (wiki)"
```

Comment:
Please don't mix unrelated changes in one pull request - stick to making the wiki changes in this PR and if you want to propose reordering the tests then do that separately.